boards/common/nrf52/Makefile.include

```diff
@@ -9,6 +9,7 @@ include $(RIOTMAKE)/tools/serial.inc.mk
 # The following configuration is dependencies specific
 # but they are resolved later
 # Hack to know now if 'nordic_softdevice_ble' is used
+-include $(APPDIR)/Makefile.board.dep
```

Comment:
Maybe you could add a bit more details on this in the comment. And also add a remark that this work around can be retiered when depricated softdevice BLE is dropped?

Reply:
Addressed

Reply:
Added the comment in the commit as well.